packages/vitepress-plugin-git-changelog/src/vite/git.ts

```diff
@@ -49,6 +49,7 @@ async function aggregateCommit(
   getReleaseTagsURL: CommitToStringsHandler,
   log: Commit,
   includeDirs: string[] = [],
+  includeExtensions: `.${string}`[] = [],
```

Comment:
Never though TypeScript can be written like this... thanks for teaching me this!